omnigan/data.py

```diff
@@ -183,10 +189,10 @@ def check_samples(self):
                 assert Path(v).exists(), f"{k} {v} does not exist"
 
 
-def get_loader(domain, mode, opts):
+def get_loader(mode, domain, opts):
```

Comment:
do you have an OCD? You didn't seem like that kind of guy

Reply:
Lmao for the record, if you look at the code in master there's a mismatch:
`def get_loader(domain, mode, opts):
    return DataLoader(
        OmniListDataset(
            domain, mode, opts, transform=transforms.Compose(get_transforms(opts))
        ),
        batch_size=opts.data.loaders.get("batch_size", 4),
        # shuffle=opts.data.loaders.get("shuffle", True),
        shuffle=True,
        num_workers=opts.data.loaders.get("num_workers", 8),
    )


def get_all_loaders(opts):
    loaders = {}
    for mode in ["train", "val"]:
        loaders[mode] = {}
        for domain in ["rf", "rn", "sf", "sn"]:
            if mode in opts.data.files:
                if domain in opts.data.files[mode]:
                    loaders[mode][domain] = get_loader(mode, domain, opts)
    return loaders`

Reply:
haha ok my bad good catch :)